features/providers/change_email.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Change applicant email address
+  @javascript @vcr
+  Scenario: I want to change the email address from the about financial assessment page
+    Given I complete the journey as far as check your answers
+    Then I click 'Save and continue'
+    Then I should be on a page showing "DWP records show that your client does not receive a passporting benefit – is this correct?"
+    Then I choose 'Yes'
+    Then I click 'Save and continue'
+    And I should be on a page showing "What is your client's employment status?"
+    And I select "None of the above"
+    When I click 'Save and continue'
+    Then I should be on a page with title "We need your client's bank statements from the last 3 months"
+    Then I choose 'Yes'
+    Then I click 'Save and continue'
+    Then I should be on a page showing 'What your client has to do'
+    Then I click link 'Continue'
+    Then I should be on a page showing "Enter your client's email address"
+    Then I fill 'email' with 'test@test.com'
```

Comment:
I would change the first email e.g. `test@bad.com`
So when we change it at line 23 we expect to see the correct value at line 26 
The description says change email, but we don't actually change it! 😄

Reply:
This is a good point! I changed it in the second test but not the first - have changed it now :)